### PR TITLE
Remove apex containers from rolling

### PR DESCRIPTION
**Pull request by carlossvg**
Removing apex_containers from rolling since it is no longer maintained.

### CODE REVIEW

**Comment by quarkytale** [comment]
@carlossvg Can you please confirm if you were the maintainer or have approval from previous maintainers? The `package.xml` seems to have a general Apex.ai email as maintainer.

**Comment by carlossvg** [comment]
> @carlossvg Can you please confirm if you were the maintainer or have approval from previous maintainers? The `package.xml` seems to have a general Apex.ai email as maintainer.

I confirm I'm one of the maintainers. 

You can verify it by checking the PR where this package was added https://github.com/ros/rosdistro/pull/29026 and the list of maintainers in the upstream repo: https://gitlab.com/ApexAI/apex_containers/-/project_members

**Review by quarkytale — APPROVED** [review]

**Pull request merged**
